Tests/BootCamp.Chapter.Tests/Input/ClassData/UnequalCredentialPairs.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+namespace BootCamp.Chapter.Tests.Input.ClassData
+{
+    public class UnequalCredentialPairs : IEnumerable<object[]>
+    {
+        public IEnumerator<object[]> GetEnumerator()
+        {
+            var credentials = GenerateCredentials();
+
+            for (var index1 = 0; index1 < credentials.Count; index1++)
+            {
+                var credential1 = credentials[index1];
+                for (var index2 = 1; index2 < credentials.Count; index2++)
+                {
+                    if (index1 == index2) continue;
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5768165)